app/src/main/kotlin/batect/docker/DockerContainerCreationRequestFactory.kt

```diff
@@ -51,6 +52,7 @@ class DockerContainerCreationRequestFactory(
             image,
             network,
             if (command != null) command.parsedCommand else emptyList(),
+            if (entrypoint != null) entrypoint.parsedCommand else emptyList(),
```

Comment:
We might have to do something slightly different here - empty list for an entrypoint means 'use system default' instead of 'use image default' (source: https://docs.docker.com/engine/api/v1.37/#operation/ContainerCreate)